packages/api/lib/writeRecords/write-granules.js

```diff
@@ -453,28 +459,33 @@ const _writeGranuleFiles = async ({
 const _writeGranuleRecords = async (params) => {
   const {
     postgresGranuleRecord,
-    apiGranuleRecord,
     knex,
     esClient = await Search.es(),
     granuleModel,
     executionCumulusId,
     granulePgModel,
     writeConstraints = true,
   } = params;
+  let { apiGranuleRecord } = params;
```

Comment:
Why is this outside of the assignment above?

Reply:
Because of the potential reassignment below. Will respond to that comment, and this change can depend on how that turns out.

Reply:
Yeah, I see this needed to not be `const`. I think we can get rid of the reassignment though. See the below comment.

Reply:
I would assume the reason it still exists is 

```
    if (!apiGranuleRecord.createdAt) {
        apiGranuleRecord.createdAt = pgGranule.created_at.getTime();
      }
```

just needs to be removed.   That block was added to ensure created_at is consistent if for some reason a default value is returned from postgres, this update tidily changes that by running another translation. 

I'm a little concerned about *adding* a translation here, given we're doing so at the top level to generate what *was* the function parameter here though.

Reply:
There is another comment thread where the actual changes are, but since the code is quoted here and the translation is referenced here, I think I should respond here. I've made some changes to the additional translation, moving the quoted code to be with that. I understand your concern about adding another translation @Jkovarik, do you have a better suggestion to how to keep the records in sync? I've not been able to come up with one yet.